storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -919,7 +919,7 @@ struct my_option xb_client_options[] = {
      0, 0, 0, 0, 0},
 
     {"no-lock", OPT_NO_LOCK,
-     "Use this option to disable table lock "
+     "Use this option to disable lock-ddl and table lock "
      "with \"FLUSH TABLES WITH READ LOCK\". Use it only if ALL your "
```

Comment:
Do we still have the possibility to execute FTWRL on 8.0 ?

Reply:
> Do we still have the possibility to execute FTWRL on 8.0 ?

if we are using --backup-locks

Reply:
I thought that with upstream 8.0 we use LOCK INSTANCE FOR BACKUP and PS8.0 we use LOCK TABLES FOR BACKUP, so on PXB8.0, there is no FTWRL. In which circumstances will we use FTWRL on 8.0?

Reply:
> I thought that with upstream 8.0 we use LOCK INSTANCE FOR BACKUP and PS8.0 we use LOCK TABLES FOR BACKUP, so on PXB8.0, there is no FTWRL. In which circumstances will we use FTWRL on 8.0?
for MyISAM tables we use FTWRL.